### PR TITLE
feat: `visyn_core/echarts` export + echarts scatter, timeline, funnel

### PR DESCRIPTION
**Pull request by dvmoritzschoefl**
- Added support for a couple of basic chart types to useChart (scatter, timeline, funnel) which are needed in aevidence
- Added typed for events that are not officially typed (like brushselected, brush, etc)
- The event params are unknown and need to be typed in the corresponding event handler

### CODE REVIEW

File: src/vis/vishooks/hooks/useChart.ts [hunk 1]
```diff
@@ -1,35 +1,55 @@
...
 use([
+  SingleAxisComponent,
+  BrushComponent,
   LegendComponent,
   LineChart,
   BarChart,
+  PieChart,
+  FunnelChart,
+  SankeyChart,
+  ScatterChart,
```
Comment (thinkh) [inline src/vis/vishooks/hooks/useChart.ts:52]: How does this impact the initial bundle size and the echarts library chunk? 

You can test it with `RSDOCTOR=true yarn bundle:prod` and then navigate to the bundle size in the upper right corner.

**Review by thinkh — COMMENTED** [review]
Generally, I'm fine with the changes. However, I'd like to know the impact on the bundle size before merging it, because we can't lazy load the echarts hook.

File: src/vis/vishooks/hooks/useChart.ts [hunk 2]
```diff
@@ -57,26 +77,41 @@ type ElementEventName =
   | 'drop'
   | 'globalout';
 
+type ExoticEventName = 'brush' | 'brushEnd' | 'brushselected' | 'highlight' | 'downplay';
```
[the remark this answers is not in the document]
Comment (dvmoritzschoefl) [reply src/vis/vishooks/hooks/useChart.ts:80]: Do you have another suggestion? I think React has a similar convention with class components where some props cannot be inferred

File: src/vis/vishooks/hooks/useChart.ts (hunk 1 above)
Comment (dvmoritzschoefl) [reply src/vis/vishooks/hooks/useChart.ts:52]: 0.13 MB difference

File: src/vis/vishooks/hooks/useChart.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (thinkh) [reply src/vis/vishooks/hooks/useChart.ts:80]: No, I don't have a better suggestion. I was just wondering but I'm fine with it.

File: src/vis/vishooks/hooks/useChart.ts (hunk 1 above)
Comment (thinkh) [reply src/vis/vishooks/hooks/useChart.ts:52]: Ok. Thanks for testing.

**Review by thinkh — APPROVED** [review]
Looks good to me.

**Comment by thinkh** [comment]
@dvmoritzschoefl The node job is still failing. Could check it and fix it?

**Review by thinkh — APPROVED** [review]

**Pull request merged**